channels/jp.m3u

```diff
@@ -1,12 +1,18 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="Animation",Animax
-https://redlabmcdn.s.llnwi.net/jp01/bs14/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="Animation",Animax Japan (Indonesian Subs)
 http://210.210.155.35/dr9445/h/h02/01.m3u8
```

Comment:
Those links don't seem to work for me at the same instance.

Reply:
Shows Aniplus for now tho
![image](https://user-images.githubusercontent.com/5430116/125361694-5a908980-e376-11eb-94ba-7a4065b0c578.png)

Reply:
Very interesting... let me try again on my part, because I can't even connect to host with what I have.

Reply:
Found out : it's geo-locked.
![vlcsnap-2021-07-13-00h26m43s718](https://user-images.githubusercontent.com/30985701/125363526-0fc04300-e371-11eb-85f2-b969cd3cac97.png)

Reply:
Interesting. Indonesian stream that I can play without VPN

Reply:
Maybe it's locked for the Asian continent / diaspora... I don't know. With VPN disabled just after, it locks back.